mysite/settings.py

```diff
@@ -103,24 +112,34 @@
 ]
 
 
+
 # Internationalization
 # https://docs.djangoproject.com/en/4.0/topics/i18n/
 
-LANGUAGE_CODE = "en-us"
+LANGUAGE_CODE = "ja"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = "Asia/Tokyo"
 
 USE_I18N = True
 
 USE_TZ = True
 
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
 
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+# How to set static files
+# https://docs.djangoproject.com/ja/4.1/howto/static-files/
+
+STATIC_URL = "static/"
+
+STATICFILES_DIRS = {
+    BASE_DIR / "static",
+}

```

Comment:
これでcss反映されてましたか？
通常は以下のようにリストで設定します

```suggestion
STATICFILES_DIRS = [BASE_DIR / "static"]
```
参考
https://github.com/django/django/blob/main/django/conf/global_settings.py#L626

Reply:
反映できました。runserverでも機能できました。
<img width="551" alt="image" src="https://user-images.githubusercontent.com/44603676/207776153-69b46c8e-d3a3-4baa-9b5b-e9fc728a9d0f.png">

こちらテストになります。
<img width="523" alt="image" src="https://user-images.githubusercontent.com/44603676/207776240-7f6c2d65-9d01-4195-8e74-edf91efea170.png">